08/kth-missing-positive-number.checkpoint.md

```diff
@@ -218,10 +212,4 @@ Space Complexity
 
 ##### !end-placeholder
 
-##### !answer
```

Comment:
This is extremely minor and not the point of this PR, but it looks like this is the only PSE where the space complexity prompt (on line 217 above) doesn't end with a `?` like the time prompt.